### ISSUE TITLE
Squiz.PHP.DisallowMultipleAssignments not ignoring typed property declarations

### ISSUE THREAD

**Issue by oojacoboo** [squizlabs/PHP_CodeSniffer#2787]
I'm getting an error on the following 7.4 compatible code:
 
```php
protected ?int $id = null;
```

**Comment by jrfnl** [comment]
Confirmed. PR #2788 should fix this.

**Comment by bkdotcom** [comment]
is this also a false positive?

`} elseif (($position = \array_search($val, $array, true)) !== false) {`
